### PR TITLE
[URLs] Add codes option to generate redeem URLs, fix typo

### PR DESCRIPTION
**Pull request by jhersh**
Thanks for another nifty CLI tool!

I added an option to generate a URL for each code that can be used to redeem that code and fixed a small typo.

### CODE REVIEW

File: bin/codes [hunk 1]
```diff
@@ -26,10 +26,11 @@ class CodesApplication
     global_option '-a', '--app_identifier STRING', 'The bundle ID to download promo codes for'
     global_option '-i', '--apple_id STRING', 'The App ID to download promo codes for'
     global_option '-o', '--output_file STRING', 'The name of the file that should be written to disk. If the file exists, the new codes will be appended.'
+    global_option '-x', '--urls', 'If specified, includes a full URL for each code that can be used to redeem that code.'
 
 
     command :download do |c|
-      c.syntax = "cert download [num]"
+      c.syntax = "codes download [num]"
```
Comment (KrauseFx) [inline bin/codes]: Ha :+1:

**Comment by KrauseFx** [comment]
Looks great, thanks for working on this @jhersh :+1: 
I'll also add a quick note to the `README`.

**Comment by jhersh** [comment]
Awesome thanks! Maybe you can push a new tag?

**Comment by KrauseFx** [comment]
Absolutely!

**Comment by KrauseFx** [comment]
Thanks! https://github.com/fastlane/codes/releases/tag/0.2.0

**Pull request merged**